src/app/app-modules/reset-password/reset-password.component.html

```
@@ -1,125 +1,123 @@
-<app-header [showRoles]="false"></app-header>
+<div class="support-wrapper">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)